docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -1011,10 +1011,8 @@ Constraint penalizeLoopedJobs(ConstraintFactory factory) {
 
 ====
 
-[IMPORTANT]
+[NOTE]
 ====
-Do not use a @ShadowVariableLooped property in a method annotated with @ShadowSources.
-@ShadowVariableLooped properties can be updated after the @ShadowSources marked method is called, causing score corruption.
 @ShadowSources marked methods do not need to check @ShadowVariableLooped properties, since they are only called if all their dependencies are not looped.
 ====
```

Comment:
```suggestion
NOTE: `@ShadowSources`-annotated methods do not need to check `@ShadowVariableLooped`-annotated properties. These methods are only called if neither of their dependencies are looped, and therefore the value of this property at that time is guaranteed to be `false`.
```

Reply:
The simplified `NOTE:` syntax works with AsciiDoc and does the same thing as the more verbose `[NOTE]`.